mptt/managers.py

```diff
@@ -794,46 +553,34 @@ def _inter_tree_move_and_close_gap(
         connection = self._get_connection(instance=node)
         qn = connection.ops.quote_name
 
-        opts = self.model._meta
         inter_tree_move_query = """
         UPDATE %(table)s
```

Comment:
The reason is simple: The code is a bit older than Django's support for Case/When :)

Case/When should work too.